### PR TITLE
Additions to folder saving for the dev branch

### PR DESCRIPTION
**Pull request by Michael-Reefe**

### CODE REVIEW

**Comment by Michael-Reefe** [comment]
Identical to #31 (and #30), but for dev

File: main/observing/calibration.py [hunk 1]
```diff
@@ -150,33 +157,38 @@ def take_darks(self, ticket):
         if not exp_times:
             logging.error('Wrong data type for exp_time(s) argument')
             return False
-        if not os.path.exists(os.path.join(self.image_directory, 'Darks_{}'.format(ticket.name))):
-            os.mkdir(os.path.join(self.image_directory, 'Darks_{}'.format(ticket.name)))
+        if not os.path.exists(os.path.join(self.image_directories[ticket], 'Darks_{}'.format(ticket.name))):
+            os.mkdir(os.path.join(self.image_directories[ticket], 'Darks_{}'.format(ticket.name)))
+        else:
+            print('Dark folder already exists!  Assuming they have been collected, & aborting dark collection...')
```
Comment (Kakon24) [inline main/observing/calibration.py]: Log

File: main/observing/calibration.py [hunk 2]
```diff
@@ -81,8 +81,14 @@ def take_flats(self, ticket):
         if not filters:
             logging.error('Wrong data type for filter(s) argument')
             return False
-        if not os.path.exists(os.path.join(self.image_directory, 'Flats_{}'.format(ticket.name))):
-            os.mkdir(os.path.join(self.image_directory, 'Flats_{}'.format(ticket.name)))
+        if not os.path.exists(os.path.join(self.image_directories[ticket], 'Flats_{}'.format(ticket.name))):
+            os.mkdir(os.path.join(self.image_directories[ticket], 'Flats_{}'.format(ticket.name)))
+        else:
+            print('Flat folder already exists!  Assuming they have been collected, & aborting flat collection.')
```
Comment (Kakon24) [inline main/observing/calibration.py]: Log

**Review by Kakon24 — CHANGES_REQUESTED** [review]
Minor changes

**Review by Kakon24 — APPROVED** [review]

**Pull request merged**